scripts/changelog.sh

```diff
@@ -101,9 +101,11 @@ function emailTemplate() {
   changelogLinksCount=$(echo -e "$changelogLinks" | wc -l)
   githubLinkId="["$((changelogLinksCount + 2))"]"
   gitterLinkId="["$((changelogLinksCount + 3))"]"
-  lines="Release announcement for Eclipse JKube $1\n\n"
-  lines+="Hi All,\n\n"
-  lines+="We are pleased to announce Eclipse JKube $1 was just released. You can find the release at Maven Central [1].\n\n"
+  lines="[ANNOUNCE] Eclipse JKube $1 released\n\n"
+  lines+="Hi Everyone,\n\n"
+  lines+="We are pleased to announce Eclipse JKube $1 was just released! You can find the release at Maven Central [1].\n\n"
+  lines+="Eclipse JKube is a collection of plugins and libraries that are used for building container images using Docker, JIB or S2I build strategies.\nEclipse JKube generates and deploys Kubernetes/OpenShift manifests at compile time too.\n\n"
+  lines+="Using this release:\n\n<plugin>\n  <groupId>org.eclipse.jkube</groupId>\n  <artifactId>kubernetes-maven-plugin</artifactId>\n  <version>$1</version\n</plugin>\n\n(Check Maven Central [1] for the rest of artifacts)\n\n"
```

Comment:
Not sure whether it should be here, but do you think we should add a section on migrating to JKube from FMP. At this point, our target audience is people currently using FMP

Reply:
It's a good idea. Let me change.

Reply:
Changed, this would be the output for the current release:
```
[ANNOUNCE] Eclipse JKube 1.0.0-rc-1 released

Hi Everyone,

We are pleased to announce Eclipse JKube 1.0.0-rc-1 was just released! You can find the release at Maven Central [1].

Eclipse JKube is a collection of plugins and libraries that are used for building container images using Docker, JIB or S2I build strategies.
Eclipse JKube generates and deploys Kubernetes/OpenShift manifests at compile time too.

JKube is the successor to the depreacted Fabric8 Maven Plugin, please check our migration guide [2] if you come from FMP.

Using this release:

<plugin>
  <groupId>org.eclipse.jkube</groupId>
  <artifactId>kubernetes-maven-plugin</artifactId>
  <version>1.0.0-rc-1</version
</plugin>

(Check Maven Central [1] for the rest of artifacts)

These are the features and fixes included in 1.0.0-rc-1:
- Fix...

Your feedback is highly appreciated, you can provide it replying to the mailing list or through the usual channels. [24] [25]

[1] https://repo1.maven.org/maven2/org/eclipse/jkube/kubernetes-maven-plugin/1.0.0-rc-1/
[2] https://www.eclipse.org/jkube/docs/migration-guide/
[3] https://github.com/eclipse/jkube/issues/...
[24] https://github.com/eclipse/jkube
[25] https://gitter.im/eclipse/jkube

```